### ISSUE TITLE
Ability to change theme

### ISSUE THREAD

**Issue by OdNairy** [Legoless/Alpha#40]
Nowadays we can change theme by manual call 

``` objective-c
[ALPHAManager defaultManager].theme = [ALPHAGreenSeaColorPalette defaultPalette].paletteTheme;
```

We need settings for ALPHA or something like that to change theme/palette.

**Comment by Legoless** [comment]
You mean an UI? If yes, I was thinking about that in the past already. But in general theme was not meant to be as a changeable property and should be chosen by the developer who integrates Alpha. I've received some feedback that Alpha menu already has many choices, so populating it with themes would just clutter it even more, while not keeping any useful app-related information.

On the other side, the theme UI will be implemented as a part of Client UI library, described in #30 .

**Comment by OdNairy** [comment]
You are right I'm talking about UI. We will be needed in settings to implement plugin load/unload functionality at least. You are right we have a lot of objects on ALPHA menu but there are still nothing on `leftBarButtomItem` slot in root Menu controller. I think we can configure settings icon there and open controller to manage `ALPHA` stuff.
We cannot use #30 as only way to change theme since it's not a required functionality.

Also i'm looking to provide ALPHA for jailbreak community to import ALPHA.dylib into any app you want and research application. In this case we cannot expect developer to change code for theme selection.

**Comment by scj643** [comment]
I personally think being able to change the coloring is what is really needed gold text on a black background isn't the most legible thing in the world and this is coming from someone who has issues reading text sometimes.

**Comment by Legoless** [comment]
@OdNairy: You are right about left bar button item being unoccupied, but I would probably add this to the main circular menu as a separate action. I'll think about it and will let you know before going to implement it.

@scj643: Hm, I must say I am a little surprised, gold on dark background does provide great contrast and should be easy to read. I would love if you would explain a bit more about why is this a problem for you. In the meantime, you can always use another theme, by setting it yourself in the code. I'll add instructions for this to the Wiki asap.

**Comment by scj643** [comment]
The reason why I say this is because I'm visually impaired and Black on white is more visible for me.

**Comment by Legoless** [comment]
Okay, understood. For now this will be the default theme, but I'll add another theme that will be black on white which you can use. And we'll make it easier to switch themes soon, so hopefully that will solve the problem for you.
